### DISCUSSION TITLE
en_core_web_sm/md/lg stopped loading today (02/04/2024)

### DISCUSSION THREAD

**Discussion by nmwhitehead** [explosion/spaCy#13333] [Help: Installation] [answered]
## How to reproduce the behaviour

nlp = spacy.load('en_core_web_sm') or spacy.load('en_core_web_md') or spacy.load('en_core_web_lg')

## Your Environment

* Operating System: Windows 10
* Python Version Used: 3.118 and 3.7.3
* spaCy Version Used: 3.7.2 and 2.3.9

When I run basic code of  ```spacy.load('en_core_web_sm')```, I get an error message ending with 
```AttributeError: 'EntryPoints' object has no attribute 'get'```
The reason I have multiple versions of both Python and spaCy listed is that upon Googling that error type, I found it to be a version based error type, so I tried fiddling around with the versions.

**Comment by svlandeg** [comment, marked as answer]
Hi! That feels like an installation error to me - I get that you tried different versions _after_ getting this error, but could you try starting fresh once more, in an isolated environment?

``` 
python -m venv .env
source .env/bin/activate
pip install -U pip setuptools wheel
pip install -U spacy
python -m spacy download en_core_web_sm

>>> import spacy
>>> nlp = spacy.load("en_core_web_sm")
```

**Reply by nmwhitehead** [reply]
Thank you! That worked like a charm

**Reply by svlandeg** [reply]
Happy  to hear it!
